src/cable/connection.cr

```diff
@@ -125,6 +125,7 @@ module Cable
 
     def unsubscribe(payload : Cable::Payload)
       if channel = Connection::CHANNELS[connection_identifier].delete(payload.identifier)
+        channel.close
         channel.unsubscribed
```

Comment:
Looks like [unsubscribed](https://github.com/cable-cr/cable/blob/master/src/cable/channel.cr#L36) already gets called in `close`, so maybe `channel.unsubscribed` can just go away? Otherwise I think this is being called twice.